### ISSUE TITLE
Required ZWJ key for Old Hungarian lajout

### ISSUE THREAD

**Issue by istszab** [keymanapp/keyboards#1676]
The ZWJ key is required for the Old Hungarian texts, for example Noto-fonts project have ~100 ligatures.
I will to do it.

**Comment by LornaSIL** [comment]
I assume you are referencing this keyboard `old_hungarian`. ZWJ is U+200D. Do you need ZWNJ (U+200C)?
What keys should they go on?
On the RALT layer there are some empty keys you could choose from: https://help.keyman.com/keyboard/old_hungarian
May RALT `=` or RALT `\`?

**Comment by istszab** [comment]
> I assume you are referencing this keyboard `old_hungarian`. ZWJ is U+200D. Do you need ZWNJ (U+200C)?
> What keys should they go on?
> On the RALT layer there are some empty keys you could choose from: https://help.keyman.com/keyboard/old_hungarian
> May RALT `=` or RALT `\`?

The ZWNJ is not necessary according to experience. On the desktop keyboard position of ZWJ is the top-left key without any modifier keys (according to xkeyboard-configs' layout). Now it is the position of § sign, but the § key has an another position, too.
On the layout of phones required change the space-line for all level to the following:
[ZWJ key on phone layout's spaceline](https://github.com/keymanapp/keyboards/files/7243073/old_hungarian_ZWJ_on_spaceline.pdf)

**Comment by istszab** [comment]
@LornaSIL Why did you asked https://github.com/keymanapp/keyboards/issues/1676#issuecomment-928158533 from me?
Do you think it's better, if you do it?
In that case my answer is https://github.com/keymanapp/keyboards/issues/1676#issuecomment-929061120
If not, please drop a message for me.

**Comment by LornaSIL** [comment]
I had created the first Keyman versions so I assumed I would need to do it. If you feel confident in doing it, I'm happy for you to do so!
Do remember to bump the keyboard version number. Thanks!

**Comment by istszab** [comment]
@LornaSIL  Is it possible, that on the touch keyboard appear **"Lig" instead of "ZWJ"** (Hungarian: "Ligatúrákhoz" = "For ligatures"),
**"JB" instead of "RLM"** (_Hungarian: "Jobbról-balra" = "Right to left"_) 
and **"BJ" instead of "LRM"** (_Hungarian: "Balról-jobbra" = "Left to right_")?
_**These layouts will be used by Hungarians frequently.**_

**Comment by LornaSIL** [comment]
Yes, you could do that. It would be helpful to update the welcome.htm and the help .php file to document what `Lig`, `JB`, and `BJ` mean.  Feel free to add English and Hungarian descriptions of the meaning. Several keyboards include documentation in two languages. It doesn't have to be extensive just something like:
"_Zero Width Joiner_ is available on the key that says **Lig** (Ligatúrákhoz), the _Right to left mark_ to available on the key that says **JB** (Jobbról-balra) and the _Left to right mark_ is available on the key that says **BJ** (Balról-jobbra)".

**Comment by istszab** [comment]
@LornaSIL It's almost done (It's work). Should I upload binary files - keyboards picture in png - through the git, or that will be generated by leader developer?

**Comment by LornaSIL** [comment]
What I need are the source files (not compiled). If you could zip everything up and put it in dropbox or Google drive and send me the link I could submit it for you.

**Comment by istszab** [comment]
@LornaSIL 
If you test the layout on Windows, MS Office and MS Edge software never will be working. The MS blocks the Old Hungarian standard ability. They say to András Tisza, that MS loses too much on the Hungarian market. They answered to András, if he wants to unlock the block, give them a large amount of money. Therefore, Viktor Kovács asked the developers of the LibreOffice project for LibreOffice to support the use of the "Old Hungarian" standard. It's the "hidden" support, the CTL language must be set as "none".
